Automattic/wp-calypso
# Stats: move datepicker and interval dropdown to `client/components`

## Proposed Changes

Move date picker and interval dropdown to `client/components`, so that we could share easily across the company in the future. We'll be tackling the name changes in follow up PRs to limit the scope.

## Testing Instructions

* Open Calypso Live Branch (still need feature flag)
* Ensure date picker works

## Pre-merge Checklist
